components/wysiwyg/extensions/codeblock/codeblock.component.tsx

```diff
@@ -0,0 +1,234 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {useCallback, useEffect, useState} from 'react';
+import classNames from 'classnames';
+import styled from 'styled-components';
+import {CSSTransition} from 'react-transition-group';
+import {offset, useFloating, size} from '@floating-ui/react-dom';
+import {ChevronDownIcon} from '@mattermost/compass-icons/components';
+import {NodeViewContent, NodeViewWrapper} from '@tiptap/react';
+import {NodeViewProps} from '@tiptap/core/src/types';
+
+import {DropdownContainer} from '../../components/toolbar/toolbar_controls';
+import {useGetLatest} from '../../components/toolbar/toolbar_hooks';
```

Comment:
access through `@components/toolbar/...`

Reply:
this `components` is a sub-folder inside the `wysiwyg` folder, so this change would break it